components/infobox/wikis/wildrift/infobox_team_custom.lua

```diff
@@ -0,0 +1,100 @@
+---
+-- @Liquipedia
+-- wiki=wildrift
+-- page=Module:Infobox/Team/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local Class = require('Module:Class')
+local Lua = require('Module:Lua')
+local String = require('Module:StringUtils')
+local Template = require('Module:Template')
+local Variables = require('Module:Variables')
+
+local Injector = Lua.import('Module:Infobox/Widget/Injector', {requireDevIfEnabled = true})
+local Team = Lua.import('Module:Infobox/Team', {requireDevIfEnabled = true})
+
+local Widgets = require('Module:Infobox/Widget/All')
+local Cell = Widgets.Cell
+
+local CustomTeam = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _team
+
+function CustomTeam.run(frame)
+	local team = Team(frame)
+	_team = team
+	_args = _team.args
+
+	-- Automatic org people
+	team.args.coach = Template.expandTemplate(frame, 'Coach of')
+	team.args.manager = Template.expandTemplate(frame, 'Manager of')
+	team.args.captain = Template.expandTemplate(frame, 'Captain of')
+
+	team.createWidgetInjector = CustomTeam.createWidgetInjector
+	team.createBottomContent = CustomTeam.createBottomContent
+	team.addToLpdb = CustomTeam.addToLpdb
+	team.getWikiCategories = CustomTeam.getWikiCategories
+	return team:createInfobox(frame)
+end
+
+function CustomTeam:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomTeam:createBottomContent()
+	return Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing matches of',
+		{team = _team.name}
+	) .. Template.expandTemplate(
+		mw.getCurrentFrame(),
+		'Upcoming and ongoing tournaments of',
+		{team = _team.name}
+	)
+end
```

Comment:
Template based infobox only display `Template:Placement summary`, not the two mentioned here

Reply:
Should I keep it? 
![image](https://user-images.githubusercontent.com/88981446/191935236-bfc9bfda-7312-4fea-9ce0-176f08774ed9.png)

I have edit the template call that only placement summary would be called. 

With the upcoming match/tournament are through the module instead (team pages originally had the **{{Upcoming_and_ongoing_matches_of}}** being type separately so currently theres a duplicate set of it, but I'm planned to bot all in here out https://liquipedia.net/wildrift/index.php?title=Special:Search&limit=250&offset=0&profile=default&search=insource%3A%27%7B%7BUpcoming_and_ongoing_matches_of%7D%7D%3Cbr%3E%27)

Reply:
Put the Placement Summary inside the module instead of the template